libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -10,6 +10,49 @@
 
 namespace caf::net::web_socket {
 
+// -- static utility functions -------------------------------------------------
+
+expected<std::pair<uint16_t, const_byte_span>>
+framing::decode_closing_payload(const_byte_span payload) {
+  if (payload.size() == 0)
+    return std::make_pair(0, payload);
+  if (payload.size() < 2)
+    return make_error(caf::sec::protocol_error,
```

Comment:
This is currently not covered by the tests. Mind adding an additional test that sends a single byte to cover this branch as well?

Reply:
Added.